examples/testing/main_test.go

```diff
@@ -66,3 +66,15 @@ func TestIntMinTableDriven(t *testing.T) {
 		})
 	}
 }
+
+// Benchmark tests typically go in `_test.go` files and are
+// named beginning with `Benchmark`. The `testing` runner
+// executes each benchmark function several times, increasing
+// `b.N` on each run until it collects a precise measurement.
+func BenchmarkIntMin(b *testing.B) {
+	// To measure a benchmark, the `testing` runner calls the
+	// function in a loop `b.N` times.
```

Comment:
Hmm, this doesn't sound right either, sorry. The runner doesn't call our function in a loop - it's our code that does it.

I would say something like: "Typically the benchmark runs a function we're benchmarking in a loop `b.N` times"